app/src/components/ProjectView/ProjectView.js

```diff
@@ -39,11 +39,15 @@ export function generateNewNode(x, y, input_label) {
 }
 
 // Construct a some sample nodes:
-const node0 = generateNewNode(700,100, 'PROJECT-X')
+export const root = generateNewNode(700,100, 'PROJECT-X')
```

Comment:
Brauchst du root irgendwo? Das würde nämlich eigentlich in die database gehören

Reply:
nein, danke fürs drauf hinweisen, das hatte ich nur zum ausprobieren gebraucht